src/core/WorkerPool.ts

```diff
@@ -104,9 +104,9 @@ class WorkerPool {
     const info = this.runInfo[infoIndex]
 
     if (info?.canceled === true) {
-      this.clearTask(info.index)
       // @ts-expect-error: error TS2722: Cannot invoke an object which is
       info?.reject('Remaining tasks canceled')
+      this.clearTask(info.index)
```

Comment:
Do you need `info?.index`?

Reply:
The `?` on `info?.reject` was not needed :-) -- removed now.